modulefiles/srw_common.lua

```diff
@@ -1,10 +1,10 @@
-load_any("jasper/2.0.25","jasper/2.0.32")
-load_any("zlib/1.2.11","zlib/1.2.13")
+load_any("jasper")
```

Comment:
@natalie-perlin  could you please check if we can keep particular library numbers

Reply:
@RatkoVasic-NOAA - reverting the changes. I've ran it both ways, and both ways work on GCP and Azure. AWS is still an issue in my runs, regardless of the srw_common.lua; the issue could be with my bash environment.